.github/workflows/docker-pull.yml

```diff
@@ -0,0 +1,39 @@
+name: docker-pull
+
+on:
+  schedule:
+    - cron: 0 0 1 * *
+
+jobs:
+  docker-pull:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Pull images
+        run: |
+          lastrepo=""
+          lastseen=""
+          images=()
+
+          for image in $(cat legacy-docker-images.txt); do
+              images+=("${image}")
+          done
+
+          for module in tools/*; do
+              if [ -f "$module"/Dockerfile ]; then
+                  images+=($(echo "luslab/nf-modules-${module##*/}:$(cat "$module"/VERSION)"))
+              fi
+          done
+
+          for image in "${images[@]}"; do
+              # Parse repository name, we will remove the previous image if different
+              readarray -d : -t repository <<< "$image"
+              if [ "${lastseen}" != "" ] && [ "${repository[0]}" != "${lastseen}" ]; then
+                  printf "Removing image ${lastseen} to make room."
+                  docker rmi "${lastrepo}" || docker rmi "${lastseen}"
+              fi
```

Comment:
Do we need this? Is this code to remove an image with a previous version? If so, lets just leave on docker and wait for it to be removed.

Reply:
it doesnt remove it from docker. it removes the pulled image from the github system so that it doesnt get clogged up if we are pulling lots of images

Reply:
ahhhh ok, good thinking!